polygeist/lib/polygeist/Ops.cpp

```diff
@@ -1481,8 +1481,6 @@ class MetaPointer2Memref final : public OpRewritePattern<Op> {
 
   Value computeIndex(Op op, size_t idx, PatternRewriter &rewriter) const;
 
-  void rewrite(Op op, Value ptr, PatternRewriter &rewriter) const;
```

Comment:
Renamed because this member function hides a virtual function in the base class.